### PR TITLE
Refactor all all operator linebreaks  to linebreaks then operators

### PR DESCRIPTION
**Pull request by AllenBW**
## Before
screen shot 2016-09-21 at 9 03 33 am

## After
screen shot 2016-09-21 at 9 02 46 am

### CODE REVIEW

File: client/app/services/designer-state.service.js [hunk 1]
```diff
@@ -174,17 +174,17 @@
         for (var i = 0; i < toolboxTabs.length; i++) {
           if (toolboxTabs[i].subtabs) {
             for (var s = 0; s < toolboxTabs[i].subtabs.length; s++) {
-              if (toolboxTabs[i].title.toLowerCase() === 'network' &&
-                toolboxTabs[i].subtabs[s].title.toLowerCase() === 'generic' &&
-                genericSubType === 'load_balancer') {
+              if (toolboxTabs[i].title.toLowerCase() === 'network'
+                && toolboxTabs[i].subtabs[s].title.toLowerCase() === 'generic'
+                && genericSubType === 'load_balancer') {
                 return toolboxTabs[i].subtabs[s];
```
Comment (dtaylor113) [inline client/app/services/designer-state.service.js]: This code is removed in my latest/open PR

File: client/app/services/designer-state.service.js [hunk 2]
```diff
@@ -174,17 +174,17 @@
...
-                toolboxTabs[i].preTitle.toLowerCase() === 'compute' &&
-                toolboxTabs[i].title.toLowerCase() === 'infrastructure' &&
-                toolboxTabs[i].subtabs[s].title.toLowerCase() === 'generic' &&
-                genericSubType === 'vm') {
+              if (toolboxTabs[i].preTitle
+                && toolboxTabs[i].preTitle.toLowerCase() === 'compute'
+                && toolboxTabs[i].title.toLowerCase() === 'infrastructure'
+                && toolboxTabs[i].subtabs[s].title.toLowerCase() === 'generic'
+                && genericSubType === 'vm') {
                 return toolboxTabs[i].subtabs[s];
```
Comment (dtaylor113) [inline client/app/services/designer-state.service.js]: This too is removed in latest/open PR

**Comment by dtaylor113** [comment]
Hi @AllenBW , great work on getting the code in shape! :+1:   
It would be great if you could somehow indicate what rule is being addressed in these PRs.  In this case 'operator-linebreak'.   
How can I view the charts you are generating for this PR?  Is it code climate or eslint?  -thanks!

**Comment by AllenBW** [comment]
@dtaylor113  rule being addressed is in pr title and commit message, can add to body as well if you would like

I used https://github.com/IanVS/eslint-nibble to break down eslint results, it helps where there are hundreds of issues

**Pull request merged**